google-cloud-pubsub-v1/lib/google/cloud/pubsub/v1/subscription_admin.rb

```diff
@@ -37,15 +37,17 @@ module V1
         #
         # @example Load this service and instantiate a gRPC client
         #
-        #     require "google/cloud/pubsub/v1/subscriber"
-        #     client = ::Google::Cloud::PubSub::V1::Subscriber::Client.new
+        #     require "google/cloud/pubsub/v1/subscription_admin"
+        #     client = ::Google::Cloud::PubSub::V1::SubscriptionAdmin::Client.new
         #
-        module Subscriber
+        module SubscriptionAdmin
         end
+
+        Subscriber = SubscriptionAdmin
```

Comment:
This alias should go into subscription_admin/helper.rb so that it doesn't get overwritten when the file is regenerated. Same for the one in topic_admin.